package.json

```diff
@@ -34,7 +34,8 @@
     "request": "^2.88.2",
     "settings-sharelatex": "^1.1.0",
     "socket.io": "https://github.com/overleaf/socket.io/archive/0.9.19-overleaf-4.tar.gz",
-    "socket.io-client": "https://github.com/overleaf/socket.io-client/archive/0.9.17-overleaf-3.tar.gz"
+    "socket.io-client": "https://github.com/overleaf/socket.io-client/archive/0.9.17-overleaf-3.tar.gz",
+    "underscore": "1.7.0"
```

Comment:
Would this be better as `"^1.7.0"`?

Reply:
In case we want to upgrade it, yes. But I do not feel like doing so at this time, there is already a lot in queue with the deploy of the new redis version.